examples/zkapps/01-hello-world/README.md

```diff
@@ -1,26 +1,39 @@
 # Mina zkApp: 01 Hello World
 
-This template uses TypeScript.
+This Hello World tutorial helps you get started with o1js, zkApps, and programming with zero-knowledge proofs.
 
-## How to build
+## Version:
+- o1js: **0.15.2**
+
+## Tutorial
+
+For step by step tutorial, you can follow this [link](https://docs.minaprotocol.com/zkapps/tutorials/hello-world).
+
+## How to compile/build the project code:
 
 ```sh
 npm run build
 ```
 
-## How to run tests
+## How to run compiled code:
```

Comment:
@LuffySama-Dev by now you know that I am not a developer, thank you for being a developer who cares about docs! 

thanks for removing these steps for tests, since Tutorial 1 does not include test files! You are on fire, thank you

Reply:
Hi @barriebyron ,

Thank you so much!!  😃